app/dao/returned_letters_dao.py

```diff
@@ -50,6 +51,25 @@ def insert_or_update_returned_letters(references):
         db.session.connection().execute(stmt)
 
 
+def fetch_returned_letter_count(service_id):
```

Comment:
Wonder if function name should be `fetch_recent_returned_letter_count` to indicate the behaviour with only finding things reported in the last 7 days

Reply:
Changed in 1c058e07